bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/graphics/FontMetrics.java

```diff
@@ -31,6 +31,16 @@ public final class FontMetrics {
 FontMetrics() {
 }
 
+public FontMetrics clone () {
```

Comment:
Same here, I believe `@since 3.125` is missing

Reply:
Confirmed, it is needed.

Reply:
done